### PR TITLE
[fix] fix failing doctests in requests/*

### PR DESCRIPTION
**Pull request by aless10**
## Why this PR
Running the command:
`
$ py.test --doctest-module requests
`
gives failing doctests. This PR aims to fix these failing doctests.

Issue Link: https://github.com/kennethreitz/requests/issues/4986

## How to test
`
$ py.test --doctest-module requests
`
All the 11 doctests should pass.

### CODE REVIEW

File: requests/__init__.py [hunk 1]
```diff
@@ -9,29 +9,22 @@
...
-   >>> print(r.text)
-   {
-     ...
-     "form": {
-       "key2": "value2",
-       "key1": "value1"
-     },
-     ...
-   }
+   >>> r.json()
```
Comment (luckydenis) [inline requests/__init__.py]: Hi, I would suggest to do so: 
```
   >>> print(r.text)
   ... # doctest: +ELLIPSIS
   ... # doctest: +NORMALIZE_WHITESPACE
   {
     ...
     "form": {
       "key2": "value2",
       "key1": "value1"
     },
     ...
   }
```
So we change the test minimally.

Comment (alessio-izzo) [reply requests/__init__.py]: Thanks. I did not know that I could use this syntax.

Comment (aless10) [reply requests/__init__.py]: Thanks. I did not know that I could use this syntax. I'm going to change it according to your suggestion.

Comment (aless10) [reply requests/__init__.py]: I added NORMALIZE_WHITESPACE and ELLIPSIS to the pytest.ini file and removed from the doctest in order to use them in pytest configuration.

**Comment by kennethreitz** [comment]
fantastic

**Pull request merged**